model_analyzer/config/input/config.py

```diff
@@ -255,10 +254,53 @@ def objective_list_output_mapper(objectives):
                 'concurrency',
                 flags=['-c', '--concurrency'],
                 field_type=ConfigListNumeric(int),
-                default_value=1,
                 description=
                 "Comma-delimited list of concurrency values or ranges <start:end:step>"
                 " to be used during profiling"))
+        self._add_config(
+            ConfigField('perf_analyzer_timeout',
+                        field_type=ConfigPrimitive(int),
+                        default_value=600,
+                        description="Perf analyzer timeout value in seconds."))
+        self._add_config(
+            ConfigField(
+                'perf_analyzer_cpu_util',
+                field_type=ConfigPrimitive(float),
+                default_value=40,
+                description=
+                "Maximum CPU utilization value allowed for the perf_analyzer.")
+        )
+        self._add_config(
+            ConfigField(
+                'run_config_search_max_concurrency',
```

Comment:
Do you think we can make these a config section? Like `run_config_search_parameters`

Reply:
We may want to add this CLI and it will be easier to interact with them if they are at the top level.